html/views/wb/tcdg2016wb.js

```diff
@@ -123,10 +128,10 @@ function makeSkaterRows(t, id, number) { //team, id, number
 	pd.append($('<div width=10%>').addClass('Number').text(number));
 	pd.append($('<div width=50%>').addClass('Name').text(WS.state['ScoreBoard.Team(' + t + ').Skater(' + id + ').Name']));
 	$.each([1, 2, 3, 4, 5, 6, 7, 8, 9], function(idx, c) {
-		pd.append($('<div width=10%>').addClass('Box Box' + c).html('&nbsp;'));
+		pd.append($('<div width=10%>').addClass('Box Box' + c).html('&nbsp;').css('color',teamFColor));
 	});
-	pd.append($('<div>').addClass('BoxFO_EXP').html('&nbsp;'));
-	pd.append($('<div>').addClass('Total').text('0'));
+	pd.append($('<div>').addClass('BoxFO_EXP').html('&nbsp;').css('color',teamFColor));
+	pd.append($('<div>').addClass('Total').text('0').css('color',teamFColor));
```

Comment:
Is this the best way to handle this?

The existing colour handling code is up around line 38, and we should try to keep with that.

Reply:
Could you use your approach from https://github.com/rollerderby/scoreboard/blob/dev/html/views/standard/index.js#L87 ?

Reply:
Looking through the other dashboards, we appear to have a mix of techniques for this.

We should probably look at adding a utility function to manage the css classes for this, the XML side of the house has something vaguely in that area.